engine/apps/labels/models.py

```diff
@@ -44,7 +45,7 @@ def get_or_create_by_name(cls, organization: "Organization", key_name: str) -> t
             label, _ = LabelsAPIClient(organization.grafana_url, organization.api_token).get_label_by_key_name(
                 label_key
             )
-        except LabelsRepoAPIException as e:
+        except (LabelsRepoAPIException, JSONDecodeError) as e:
```

Comment:
What exact case it's handling?

Reply:
When we get empty response from label repo (not sure why it happens)